_extras/e02-jekyll-lesson-example.md

```diff
@@ -58,21 +57,12 @@ _config.yml		bin
 
 You can now request that a container is created that will compile the files in this set into the lesson website, and will run a simple webserver to allow you to view your version of the website locally. Note that this command will be long and fiddly to type, so you probably want to copy-and-paste it into your shell window. This command will continue to (re-)generate and serve up your version of the lesson website, so you will not get your shell prompt back until you type <kbd>control</kbd>+<kbd>c</kbd>. This will stop the webserver, since it cleans away the container.
 
-> ## If you happen to have the `make` tool already installed...
-> We are taking an atypical approach in using the command that follows, since you are not required to have set up the `make` tool on your computer. However you may want to see whether you happen to have `make` installed anyway, by typing `make docker-serve` instead of the command below. At worst, this will fail and you can use the command shown below.
-{: .callout}
-
 For macOS, Linux and PowerShell:
 ~~~
 $ docker run --rm -it -v ${PWD}:/srv/jekyll -p 127.0.0.1:4000:4000 jekyll/jekyll:pages jekyll serve
 ~~~
 {: .language-bash}
 
-For `cmd.exe` shells on Microsoft Windows:
-~~~
-> docker run --rm -it -v "%CD%":/srv/jekyll -p 127.0.0.1:4000:4000 jekyll/jekyll:pages jekyll serve
-~~~
-
```

Comment:
Did you mean to delete this option?

Reply:
I removed as we do not use the `cmd.exe` syntax anywhere else in the lesson so it seemed confusing. Happy to leave in, if you prefer. Just let me know what you want to do here.

Reply:
Is it only here because this the only place where the command would differ for `cmd.exe`?  If there are other places it would differ that we don't cover, I'm good with removing it.